app/helpers/editor_helper.rb

```diff
@@ -0,0 +1,13 @@
+module EditorHelper
+  def editor_options
+    { :modes => ['Text', 'Json', 'Ruby', 'Html_ruby', 'Sh', 'Xml', 'Yaml'],
+          :keyBindings => ['Default', 'Emacs', 'Vim'],
+          :themes => ['Github', 'Monokai'] }
+  end
+
+  def template_name_attribute(template)
+    return 'provisioning_template[template]' if template == 'ProvisioningTemplate'
+    return 'report_template[template]' if template == 'ReportTemplate'
```

Comment:
I believe this can be extracted from the object itself,  perhaps using https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-fields_for ?